man/systemd.service.xml

```diff
@@ -563,6 +563,31 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>TimeoutAbortSec=</varname></term>
```

Comment:
so this is about putting a time limit on coredumping, right? Should that time limit really be configurable per unit? wouldn't a global setting for this suffice?

Reply:
We use systemd in an IVI system. If there is a crash you want to make sure the reboot happens soonish. Most services have small timeouts but e.g. the navigation process has a long timeout because the core file is multi-GB big.

It's also consistent with how the other timeouts (start/stop) are configured. I would like to keep that pattern.

Reply:
ok